pkg/builder/controller.go

```diff
@@ -125,6 +126,14 @@ func (blder *Builder) WithEventFilter(p predicate.Predicate) *Builder {
 	return blder
 }
 
+// WithControllerOptions overrides the controller options use in doController. Defaults to empty.
+//
+// Deprecated: Use ControllerManagedBy(Manager) and this isn't needed.
```

Comment:
why is this immediately deprecated?  It's unclear why this isn't needed with `ControllerManagedBy`...

Reply:
Sorry, copy pasted the comments and tried changing it and accidently left it there. Removed.